float32 time
# joint torques
float32 tau1
float32 tau2
float32 tau3
float32 tau4
float32 tau5
float32 tau6